### DISCUSSION TITLE
Use Prisma only for selected tables

### DISCUSSION THREAD

**Discussion by DanielStoehr** [prisma/prisma#11239] [Q&A] [answered]
Hello,

I´m new to Prisma and I want to integrate it in my NextJS App.
My special situation is that my App is using only a couple tables of the whole database.
And if someone changes tables what has nothing to do with my app and I run my migration commands I get an error message and prisma wants to reset my database.

How can I solve this issue?
Is my strategy to do that maybe completely false?

Greets
Daniel

**Comment by austincrim** [comment, marked as answer]
Hey @Brainiac-1989 👋 ,

It is not currently possible to introspect only certain tables from a DB, but I would encourage you to look through [this issue](https://github.com/prisma/prisma/issues/807) and see if any of the workarounds are suitable for your scenario.

I will call out [this strategy specifically](https://www.prisma.io/docs/concepts/components/introspection#introspecting-only-a-subset-of-your-database-schema) that involves creating a new user in the database that only has access to a subset of tables.

Let me know if you have questions!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
